src/turbomind/models/llama/SequenceManager.cc

```diff
@@ -202,20 +215,31 @@ struct Transaction {
     int allocate_{};
     int evict_{};
     int preempt_{};
+    int cache_evict_{};
+
+    int max_block_count_;
 
     Sequences victims_;
 
     const Sequences& sequences_;
     Schedule&        schedule_;
 
-    explicit Transaction(const Sequences& sequences, int index, int block_count, int input_count, Schedule& sched):
-        sequences_(sequences), schedule_(sched), index_(index), block_count_(block_count), input_count_(input_count)
+    std::shared_ptr<BlockTrie> block_trie_;
+
+    explicit Transaction(const Sequences& sequences,
```

Comment:
`explicit` is only effective for constructors to avoid implicit type conversion and it is only valid for class constructors with only one parameter.

Reply:
> `explicit` is only effective for constructors to avoid implicit type conversion and it is only valid for class constructors with only one parameter.

after C++11 https://en.cppreference.com/w/cpp/language/explicit it's ok